src/router.ts

```diff
@@ -105,74 +130,52 @@ const Router = {
     });
 
     // Handle proxied resources
-    app.use(APP_PREFIX + Constants.PROXY_PATH, nocache, auth,
-            this.proxyController);
+    app.use(APP_PREFIX + Constants.PROXY_PATH, nocache, auth, proxyController);
 
     // Let OAuth handle its own rendering
-    app.use(APP_PREFIX + Constants.OAUTH_PATH, nocache,
-            require('./controllers/oauth_controller'));
+    app.use(APP_PREFIX + Constants.OAUTH_PATH, nocache, OAuthController());
```

Comment:
You could probably just create a single instance of this and reuse it on line 168.

Reply:
Done